content/panorama/scripts/panorama_shop.js

```diff
@@ -440,6 +440,12 @@ function ShowItemInShop(data) {
 }
 
 function UpdateShop() {
+	var IsHUDFlipped = FindDotaHudElement("Hud").BHasClass("HUDFlipped")
```

Comment:
With `.HUDFlipped &` it takes `HUDFlipped` class from root element, so there's no need to reapply it